linux_os/guide/services/ssh/ssh_server/sshd_set_idle_timeout/rule.yml

```diff
@@ -12,7 +12,11 @@ description: |-
     follows:
     <pre>ClientAliveInterval <b><sub idref="sshd_idle_timeout_value" /></b></pre>
     The timeout <b>interval</b> is given in seconds. To have a timeout
+{{% if product in ['opensuse', 'sle11', 'sle12'] %}}
+    of 10 minutes, set <b>interval</b> to 600.
+{{% else %}}
     of 15 minutes, set <b>interval</b> to 900.
+{{% endif %}}
```

Comment:
This change is not needed, the time set here is an example. If strongly wanting this change, consolidate so that there is no SuSE customization here.